components/esmterrain/storage.cpp

```diff
@@ -155,7 +155,7 @@ namespace ESMTerrain
                                             osg::ref_ptr<osg::Vec4Array> colours)
     {
         // LOD level n means every 2^n-th vertex is kept
-        size_t increment = 1 << lodLevel;
+        size_t increment = static_cast<int64_t>(1) << lodLevel;
```

Comment:
size_t is a type specifier. I think there was a problem with the markdown formatting before, could you look at my edited comment?